public/api-v1.php

```diff
@@ -96,7 +97,12 @@
         $response = $response->withHeader('Content-Language', 'en');
         return $response;
     }
-    $response = $response->withJson(\Tancredi\Entity\Scope::getPhoneScope($mac, $this->storage, $this->logger),200,JSON_FLAGS);
+    if (array_key_exists('inherit',$query) and $query['inherit'] == 1) {
+        $results = \Tancredi\Entity\Scope::getPhoneScope($mac, $this->storage, $this->logger, TRUE);
+    } else {
+        $results = \Tancredi\Entity\Scope::getPhoneScope($mac, $this->storage, $this->logger, FALSE);
+    }
```

Comment:
Cannot see `array_key_exists() and` here :see_no_evil: 

```suggestion
    $inherit = isset($query['inherit']) && $query['inherit'] == 1;
    $results = \Tancredi\Entity\Scope::getPhoneScope($mac, $this->storage, $this->logger, $inherit);
```

Reply:
Please implement also the automated test case! :pray: